selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindText.java

```diff
@@ -506,4 +511,21 @@ public void openFileNodeByDoubleClick(String pathToFile) {
         .click();
     actionsFactory.createAction(seleniumWebDriver).doubleClick().perform();
   }
+
+  /** get number of found occurrences in a page */
+  public int getFoundOccurrencesNumberOnPage() {
+    List<String> results = new ArrayList<>(Arrays.asList(getResults().split(" ")));
```

Comment:
We don't need to convert from array to list and back here, so as split() command returns String[].
You can use this command here instead:
```javascript
return Integer.parseInt(getResults().split(" ")[0]);
```